From: Dirkjan Ochtman <dirkjan@ochtman.nl>
Date: Mon, 21 Jun 2021 11:33:39 +0200
Subject: [PATCH] Remove bitvec from default features

---
diff --git a/Cargo.toml b/Cargo.toml
index c12e163..4d81557 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -182,12 +182,16 @@ version = "^0.3"
 version = "0.9"
 
 [features]
-alloc = ["bitvec/alloc", "funty"]
-default = ["std", "bitvec", "lexical"]
+alloc = []
+std = ["alloc", "memchr/use_std"]
+bitvec-base = ["bitvec"]
+bitvec-alloc = ["bitvec/alloc", "funty"]
+bitvec-std = ["bitvec/std"]
+default = ["std", "lexical"]
 docsrs = []
 lexical = ["lexical-core"]
 regexp = ["regex"]
-std = ["alloc", "bitvec/std", "memchr/use_std"]
+
 [badges.coveralls]
 branch = "master"
 repository = "Geal/nom"
